bser/decode.go

```diff
@@ -115,11 +115,6 @@ func decodeArray(r io.Reader, dest reflect.Value) error {
 }
 
 func decodeObject(r io.Reader, dest reflect.Value) error {
-	dest, err := prep(dest, typGenericMap)
```

Comment:
this was here to initialize the dest to a generic map if nothing was passed in.  why do we need to remove?

Reply:
Good catch. I forgot to re-add it after misdiagnosing an issue with testing decode.